.github/workflows/checks_secure.yml

```diff
@@ -0,0 +1,38 @@
+name: Checks (secure)
+# These are run on base branch with read/write access.
+
+on:
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  dismiss_stale_approvals:
+    # Dismiss stale approvals for non-admins or if this PR comes from a fork.
+    runs-on: ubuntu-latest
+    # Only if another commit was added to the PR.
+    if: github.event.action == 'synchronize'
```

Comment:
Is it necessary? Isn't "synchronize" the only trigger?

Reply:
No longer necessary - it was necessary before I moved this into the checks_secure yml.